src/sqlancer/cockroachdb/CockroachDBProvider.java

```diff
@@ -285,4 +287,57 @@ public String getDBMSName() {
         return "cockroachdb";
     }
 
+    @Override
+    public String getQueryPlan(String selectStr, CockroachDBGlobalState globalState) throws Exception {
```

Comment:
It would be great to add a couple of test cases for parsing the query plans to ensure that everything still works after updating the database systems.